tests/datasets/test_querylog_processor.py

```diff
@@ -153,87 +153,3 @@ def test_simple() -> None:
         ],
         None,
     )
-
```

Comment:
need to confirm with team-search that this can go -- negative ids arent' supported in rust and it doesn't seem to cause problems